src/main/resources/db/hsqldb/populateDB.sql

```diff
@@ -47,10 +47,10 @@ INSERT INTO pets VALUES (11, 'Freddy', '2010-03-09', 5, 9);
 INSERT INTO pets VALUES (12, 'Lucky', '2010-06-24', 2, 10);
 INSERT INTO pets VALUES (13, 'Sly', '2012-06-08', 1, 10);
 
-INSERT INTO visits VALUES (1, 7, '2013-01-01', 'rabies shot');
-INSERT INTO visits VALUES (2, 8, '2013-01-02', 'rabies shot');
-INSERT INTO visits VALUES (3, 8, '2013-01-03', 'neutered');
-INSERT INTO visits VALUES (4, 7, '2013-01-04', 'spayed');
+INSERT INTO visits VALUES (1, 7, '2013-01-01', 'rabies shot', true, false, 1, false);
```

Comment:
We can insert multiple values with single statement. For example something like:

INSERT INTO {table_name} (column1, column2...) 
VALUES
(value1, value2),
(value1, value2),
 ....;

Also, why do we change only for hsql and not for mysql and postgre as well?
We can also use the auto-increment for the different tables, this way we will not need to specify the id.

But mainly, as mentioned in the general PR comment, we can use DB versioning tool